up-core-api/uprotocol/uattributes.proto

```diff
@@ -118,3 +118,34 @@ enum UPriority {
     // Network control such as Safety Critical
     UPRIORITY_CS6 = 7 [(uprotocol.ce_name) = "CS6"];
 }
+
+// The format for the data stored in the UMessage.
+enum UDataContentType {
```

Comment:
Shouldn't a field of this type be added to `UAttributes`?

Reply:
Done, rooky mistake :-)